app/mixins/setup-submit-controller.js

```diff
@@ -61,7 +63,7 @@ export default Ember.Mixin.create({
         controller.set('filePickerState', 'existing'); // In edit mode, dealing with existing project
         controller.set('existingState', 'new'); // In edit mode, only option to change file is to upload a NEW file
         controller.set('node', node);
-        controller.set('nodeTitle', node.get('title'));
+        controller.set('nodeTitle', fixSpecialChar(node.get('title')));
```

Comment:
I'm not sure this is necessary: the `fixstring` transform means that data from the server should just naturally look ok when we access it; we should not need to "fix" it further for display.

Also, in general, I'd encourage us to keep display logic in the template when feasible, eg using the template helper instead of modifying the property on the controller.